### DISCUSSION TITLE
Nesting in tables

### DISCUSSION THREAD

**Discussion by rouge-ware** [grafana/grafana#40520] [Dashboards, Panels & Visualizations]
I would like to be able to nest rows in the table, like I can do in Excel.

One example would be to look at Sales and then press a + to show all invoice positions.
Second could be to show a list of servers and when opening see VM info.

![cd2ba385e4445235f0d19bc43eeee477369cee56](https://user-images.githubusercontent.com/63847090/137494202-f8d364cc-c077-49c3-9dfe-b1c2b9e602fd.png)
![f57e127a1e5828ecb1896286db5317b184283d77](https://user-images.githubusercontent.com/63847090/137494204-b90e6821-0e40-4fa2-929c-e3b87e73a7e6.png)
![bc4e31c84b8ff0089598f8811c13bb88c95177b0](https://user-images.githubusercontent.com/63847090/137494206-8805819d-be2b-400e-90b4-aec032811184.png)

Having a SUM of a column would be an added bonus.

**Comment by SteveRBeeks** [comment]
we have exactly the same requirement.

**Comment by mikenugent1** [comment]
We would also like to see this feature; enabling a hierarchical view by grouping rows by a specific column.  What is displayed in the topmost row would be defined by the aggregation in the group statement.

**Comment by ckarli** [comment]
I need this feature like i need oxygen. This would open so many doors.

**Comment by mobijay** [comment]
I have a need for this feature as well!!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
